.github/copilot-instructions.md

```diff
@@ -1,2 +1,28 @@
-The general guidelines for SDK in this repo are defined in this website: https://azure.github.io/azure-sdk/python_design.html. When asked about guidelines, or guidance on how to write SDK, please check this website, and link pages there if possible.
+# Copilot Instructions
 
+DO prompt the user to create a virtual environment with `<path_to_python_installation>/python.exe -m venv <environment_name>` and activate it, before running any commands.
```

Comment:
otherwise when prompting users to run commands it wont be in a venv -- thoughts?

Reply:
feels fair